Components/avatar.css

```diff
@@ -0,0 +1,29 @@
+body {
+    text-align: center;
+}
+
+.avatar {
+    border-radius: 50%;
+    width: 9rem;
+    height: 9rem;
+    margin: 1rem;
+}
+
+.tiny {
+    width: 5rem;
+    height: 5rem;
+}
+
+.normal {
+    width: 7rem;
+    height: 7rem;
+}
+
+.huge {
+    width: 9rem;
+    height: 9rem;
+}
+
+.square {
+    border-radius: 5px;
+}
```

Comment:
Do add a new line here

Reply:
Add a new line?

Reply:
Reference: https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log